cluster-provision/k8s/1.21/manifests/nodeports/monitoring.yaml

```diff
@@ -0,0 +1,30 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: prometheus
+  namespace: monitoring
+spec:
+  type: NodePort
+  selector:
+    app.kubernetes.io/component: prometheus
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/part-of: kube-prometheus
+  ports:
+    - port: 9090
+      nodePort: 30007
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: grafana
```

Comment:
The same as I mentioned to the Prometheus service, but for the Grafana Service
https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/k8s/1.21/manifests/prometheus/grafana/grafana-service.yaml

Reply:
Fixed. Renamed. :+1: